lib/plugins/aws/package/compile/events/sqs.js

```diff
@@ -111,6 +119,12 @@ class AwsCompileSQSEvents {
               };
             }
 
+            if (event.sqs.scalingConfig) {
+              sqsTemplate.Properties.ScalingConfig = {
```

Comment:
@coderhglee, this doesn't seem correct CF config. Have you tested this configuration?

At least the AWS CF reference doesn't mention the `ScalingConfig` property -> https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventsourcemapping.html.

I have a notion that you've got inspired by SAM configuration, which is a more high-level thing than CloudFormation we're relying on here.

Reply:
Yes I have done testing with this configuration and it deployed fine. 

The document you shared has not been updated.

You can check the CF config scheme `CloudFormation > Registry: Activated extensions > AWS::Lambda::EventSourceMapping`.

**CF stack**
```
 "some_resources": {
    "Type": "AWS::Lambda::EventSourceMapping",
    "Properties": {
      "BatchSize": 10,
      "MaximumBatchingWindowInSeconds": 10,
      "Enabled": true,
      "ScalingConfig": {
        "MaximumConcurrency": 20
      }
    }
  },
```

![스크린샷 2023-01-16 오후 9 29 33](https://user-images.githubusercontent.com/24283191/212678525-19dc4afa-41cf-4d95-844a-060521eb8ef5.png)

Reply:
>  CloudFormation > Registry: Activated extensions > AWS::Lambda::EventSourceMapping.

@coderhglee can you pass a link to that?  Where exactly I can find this resource?

Reply:
@medikoo you can download Region resource provider schema files. 

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html 

`ScalingConfig.MaximumConcurrency` is defined in aws-lambda-eventsourcemapping.json file.